libnethack/src/allmain.c

```diff
@@ -502,12 +502,12 @@ you_moved(void)
                 u.ublesscnt--;
             iflags.botl = 1;
 
-            /* One possible result of prayer is healing.  Whether or * not you
```

Comment:
It's easier to review if you don't do random cleanup in unrelated CLs, especially in files that you aren't otherwise touching.

Also, this mistake is probably upstream, in which case you should send this as a patch upstream.